opendrift/models/basemodel.py

```diff
@@ -2043,7 +2043,7 @@ def set_fallback_values(self, refresh=False):
 
     def run(self, time_step=None, steps=None, time_step_output=None,
             duration=None, end_time=None, outfile=None, export_variables=None,
-            export_buffer_length=100, stop_on_error=False):
+            export_buffer_length=100, stop_on_error=False, move_from_land=True):
```

Comment:
`move_from_land` seems to be a duplicate of existing config option `seed:ocean_only`? Thus I believe `move_from_land` should be removed, and config should be used instead, for consistency.

Reply:
I don't remember exactly why I did this, but it was necessary to get it work, and it is different from seeding

Reply:
We should avoid more input parameters to methods like `run`, and instead control such things via the config-mechanism. There are several reasons for this, one being that web interfaces like Drifty are based generally on the config-mechanism, and there we want to avoid any method-specific tweaks that mus tbe hardcoded and kept in sync.